examples/qg/README.md

```diff
@@ -0,0 +1,29 @@
+# Table Question Generation (TableQG)
+
+The primary script is [run_qg.py](./run_qg.py).  This runs a transformer-based sequence generation pipeline.
+Before continuing below make sure you have OneQA [installed](../../README.md#Installation).
+
+## Supported datasets
+Currently supported datasets for training include:
+- WikiSQL (QG for TableQA)
+- SQuAD, SQuAD_v2 (QG for PassageQA i.e. MRC)
+
+Inference can be done on any table in particular dict format. Check this [notebook](../../notebooks/tableqg/tableqg_inference.ipynb) for more information.
```

Comment:
This link is broken. Looks like it should be `../../notebooks/qg/tableqg_inference.ipynb`